wgengine/netstack/netstack.go

```diff
@@ -33,6 +33,7 @@ import (
 	"gvisor.dev/gvisor/pkg/tcpip/transport/udp"
 	"gvisor.dev/gvisor/pkg/waiter"
 	"inet.af/netaddr"
+	"tailscale.com/client/tailscale"
```

Comment:
we shouldn't use the client from the server. That's too indirect/slow/unreliable.

we should be able to do what it does directly, without going over HTTP.